dev_mode/package.json

```diff
@@ -121,7 +121,7 @@
     "@lumino/widgets": "^1.30.0",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
-    "yjs": "^13.5.17"
+    "yjs": "next"
```

Comment:
Looks like there has been a couple of `yjs` releases since.

Is it already possible to update to `^13.5.27` for instance?

Reply:
For reference the move feature is being implemented in Yjs in this PR: https://github.com/yjs/yjs/pull/357

Reply:
The moving feature is currently only available in the `next` release which points to the Yjs alhpa release that contains experimental features.

The new move feature seems to work well so far. I collected feedback and some suggestions for changes and need another day of work to finish this and merge it into the Yjs main. I want to wait for funding and a new contract before I continue working on this.

I recommend against merging this PR as it points towards the alpha release of Yjs.